IDEAS/Buildings/Components/RectangularZoneTemplate.mo

```diff
@@ -311,7 +311,6 @@ protected
     linIntCon_a=linIntCon,
     dT_nominal_a=dT_nominal_win,
     linExtCon=linExtCon,
-    linExtRad=linExtRad,
```

Comment:
To be clear: I removed it only for windows.

Since `linExtRad=false` and we want it to be `true` for windows since otherwise non-linear algebraic loops are generated.